src/pages/RedirectFromOAuth.ts

```diff
@@ -1,17 +1,15 @@
 import { UserManager } from 'oidc-client-ts';
 import { div, img } from 'react-hyperscript-helpers';
+import { getOidcConfig } from 'src/libs/auth';
 import { useOnMount } from 'src/libs/react-utils';
 
 const RedirectFromOAuth = () => {
-  const userManager = new UserManager({
-    popup_redirect_uri: `${window.origin}/redirect-from-oauth`,
-    silent_redirect_uri: `${window.origin}/redirect-from-oauth-silent`,
-  });
+  const userManager: UserManager = new UserManager(getOidcConfig());
```

Comment:
Why did this configuration change? `getOidcConfig` redirects to Orchestration URLs instead of Terra UI URLs. I suspect this might lead to a redirect loop if those Orchestration URLs redirect to this page.

Reply:
Oooh, I was looking at `authorization_endpoint` and `token_endpoint` in `getOidcConfig` and getting them mixed up with `popup_redirect_uri` and `silent_redirect_uri` here. Sorry, my mistake. Disregard this comment.